packages/db/src/meta/pouch/adapters/index.ts

```diff
@@ -72,10 +72,16 @@ export const concretize = <C extends Collections, N extends AdapterName>(
         settings: settings || Couch.getDefaultSettings()
       };
     }
-    case "sqlite": {
+    case "sqlite":
+      const sqliteWarning =
+        "Deprecated sqlite pouchdb adapter in truffle-config";
+      debug(sqliteWarning);
+      console.warn(sqliteWarning);
+    // fall through and handle as indexeddb
```

Comment:
There could be projects that defined a sqlite adapter. Without this case, the truffle command will throw in default handler. Is this the right way to proceed? I'm not sure, so please chime in, @gnidan @davidmurdoch.

Reply:
The only change I would make is change the word "Deprecated", since it's been removed altogether.